lib/rules/sort-imports.js

```diff
@@ -112,15 +117,25 @@ module.exports = {
                 return node.specifiers[0].local.name;
             }
             return null;
+        }
 
+        /**
+         * Gets the number of lines between the given declarations.
+         * @param {ASTNode} node1 the ImportDeclaration node.
+         * @param {ASTNode} node2 the ImportDeclaration node.
+         * @returns {number} number of lines.
+         */
+        function getLinesBetweenMembers(node1, node2) {
```

Comment:
Perhaps we should rename this function to something like `getNumberOfLinesBetween` (as in function's description), because it doesn't return lines?